builds/misc/packages-release.yaml

```diff
@@ -0,0 +1,1116 @@
+    inputs:
+      ConnectedServiceName: 6ee18517-6d66-4730-bffd-2858a151bf69
+      FolderPath: $(Build.SourcesDirectory)
+      Pattern: Package-Template/update.cat,scripts/windows/setup/IotEdgeSecurityDaemon.ps1
+      signConfigType: inlineSignParams
+      CertificateId: 401
+      OpusName: Azure IoT Edge
+      OpusInfo: https://azure.microsoft.com/en-us/services/iot-edge/
+      inlineOperation: >-
+        [
+          {
+            "keyCode": "CP-229879",
+            "operationSetCode": "SigntoolSign",
+            "parameters": [
+              {
+                "parameterName": "OpusName",
+                "parameterValue": "Microsoft Windows"
+              },
+              {
+                "parameterName": "OpusInfo",
+                "parameterValue": "http://www.microsoft.com/windows"
+              },
+              {
+                "parameterName": "PageHash",
+                "parameterValue": "/NPH"
+              },
+              {
+                "parameterName": "FileDigest",
+                "parameterValue": "/fd sha256"
+              },
+              {
+                "parameterName": "TimeStamp",
+                "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+              }
+            ],
+            "toolName": "signtool.exe",
+            "toolVersion": "6.2.9304.0"
+          }
+        ]
+      SessionTimeout: 20
+  - task: PowerShell@2
+    displayName: Generate CAB package
+    inputs:
+      filePath: edgelet/build/windows/package.ps1
+      arguments: -CreateCab
+      script: >-
+        # Write your powershell commands here.
+
+
+        Write-Host "Hello World"
+
+
+        # Use the environment variables input below to pass secret variables to this script.
+  - task: CopyFiles@2
+    displayName: Copy Package Artifact Staging
+    inputs:
+      SourceFolder: .
+      Contents: '*.cab'
+      TargetFolder: $(build.artifactstagingdirectory)
+  - task: CopyFiles@2
+    displayName: Copy Setup Script to Artifact Staging
+    inputs:
+      SourceFolder: scripts\windows\setup
+      Contents: IotEdgeSecurityDaemon.ps1
+      TargetFolder: $(build.artifactstagingdirectory)
+  - task: EsrpCodeSigning@1
+    displayName: ESRP Package CodeSigning
+    inputs:
+      ConnectedServiceName: 6ee18517-6d66-4730-bffd-2858a151bf69
+      FolderPath: $(build.artifactstagingdirectory)
+      Pattern: '*.cab'
+      CertificateId: 400
+      OpusName: Azure IoT Edge
+      OpusInfo: https://azure.microsoft.com/en-us/services/iot-edge/
+      SessionTimeout: 20
+  - task: ManifestGeneratorTask@0
+    displayName: SBOM Generation Task
+    inputs:
+      BuildDropPath: $(build.artifactstagingdirectory)
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish Artifact: iotedged-windows-amd64'
+    inputs:
+      PathtoPublish: $(build.artifactstagingdirectory)
+      ArtifactName: iotedged-windows-amd64
+- job: Job_5
```

Comment:
You should remove Windows arm32; we don't support it in 1.1 LTS (notice it's disabled in the UI job).

Reply:
Done!